blazor/datagrid/custom-binding.md

```diff
@@ -988,3 +988,418 @@ The following sample code demonstrates sending additional parameters to the cust
 ```
 
 ![Passing Additional Parameters to Custom Adaptor in Blazor DataGrid](./images/blazor-datagrid-passing-additional-parameter-to-custom-adaptor.gif)
+
+## DataTable Binding
+
+DataTable represents a table with data rows and columns.
```

Comment:
Paragraph not aligned

Reply:
Hi @Venkat-Ayothi ,
Changed as per your suggestion.
Regards,
Sherlin Mini